### PR TITLE
Sending @Verdant_Jack 's Object2.png and JosephSeraph SFX

### PR DESCRIPTION
**Pull request by jetrotal**
I guess this is better now.

"Object2.png": {
            "status": "done",
            "year": "2022 ",
            "from": "RPG Maker 2000/2003",
            "authors": "Verdant_Jack CC0",
            "sources": "https://community.easyrpg.org/t/test-for-new-rtp/1067/8"
        },

### CODE REVIEW

**Review by fdelapena — APPROVED** [review]

File: AUTHORS.md [hunk 1]
```diff
@@ -5,75 +5,105 @@ Creative Commons Attribution 4.0 International license.
...
-Title/Title4.png            | (c) 2010 Francisco de la Peña [CC0]  | https://easyrpg.org
+Type|Asset Name|Year|Authors and License|Authors' URL
+:-: | :-: | :-: | :-: | :-:
+<br/>⭐<br/> ⠀ | ⠀ | ⠀ | ⠀ | ⠀
+`BattleWeapon` | Weapon.png | 2010 | russidan (Alephman) [CC-BY] | None
+<br/>⭐<br/> ⠀ | ⠀ | ⠀ | ⠀ | ⠀
+`CharSet` | Actor1.png | 2010 | Marina Navarro Travesset [CC-BY] | https://muerteatartajo.blogspot.com
+`CharSet` | Object1.png | 2017 | Tom Lemmens, <br/>Blarumyrran [CC0] (original chest) | https://github.com/lemtom
+`CharSet` | Object2.png | 2010 | Verdant_Jack [CC0] | https://community.easyrpg.org/t/test-for-new-rtp/1067/8
+<br/>⭐<br/> ⠀ | ⠀ | ⠀ | ⠀ | ⠀
```
Comment (Ghabry) [inline AUTHORS.md:16]: What is the purpose of these star symbols?

Comment (fdelapena) [reply AUTHORS.md:16]: By enabling the rich preview, it seems to be a folder separator.

Captura de pantalla de 2022-03-28 02-25-22

Maybe by making separated tables for each folder type to save a column instead is simpler?

Comment (Ghabry) [reply AUTHORS.md:16]: yeah maybe make one table per type.

**Review by Ghabry — APPROVED** [review]
I will approve this to get the assets in.
Please create another PR that "fixes" the :star: :star2:

**Pull request merged**